### ISSUE TITLE
want to add for windows as hta

### ISSUE THREAD

**Issue by MarufHasan24** [malivinayak/To-Do-Manager#47]
I want to create a windows application of you TO-Do list. Assign me on it

**Comment by malivinayak** [comment]
> windows application

What tech stack you are planning to use
elaborate & validate your plan before proceeding.

**Comment by MarufHasan24** [comment]
I'm going to use Hta and CSS as frontend and VBasic.NET and JavaScript for backend

**Comment by malivinayak** [comment]
> script files too scattered and didn't follow the rules of good coadding.

Yes, most of the contributors are beginners that's why i didn't force them about coding style
